proxy/src/main/resources/rmq.proxy.logback.xml

```diff
@@ -63,10 +63,12 @@
     <!-- Below is the logger configuration for broker-->
     <appender name="DefaultAppender"
               class="ch.qos.logback.core.rolling.RollingFileAppender">
-        <file>${user.home}/logs/rocketmqlogs/${brokerLogDir}/broker_default.log</file>
+        <file>${user.home}/logs/rocketmqlogs/${brokerLogDir:-${file.separator}}/broker_default.log</file>
```

Comment:
Why not replace all `/` with `${file.separator}`?

Reply:
resolved